rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2023-05-22"
+channel = "1.74"
```

Comment:
we need to update also the workflow files to have the same version because of the linting check

Reply:
i think fmt needs nightly to have all the options we use.